README.md

```diff
@@ -16,6 +14,21 @@ Custom Resource Definitions(CRDs) and Kubernetes-native tooling.
 
 [**Features**](#features) | [**Get started**](#get-started) | [**Documentation**](#documentation) | [**main branch builds**](#main-branch-builds) | [**Seeking help**](#seeking-help)
 
+## ✨ KIC 2.0 is in public beta! ✨
+
+We're happy to announce that Kong Ingress Controller 2.0 is in public beta.
+
+This new release comes with a major internal rearchitecture, and the following added features:
```

Comment:
```suggestion
This 2.0 beta release comes with a major internal re-architecture, and the newly added features:
```

Reply:
I don't understand why any of these changes would be helpful.

On _rearchitecture_ vs _re-architecture_: https://en.wiktionary.org/wiki/rearchitecture

Reply:
isn't 2.0 beta accurate than "new release" ? 
according to english syntax https://english.stackexchange.com/questions/325149/re-architecture, I think you mean to change the design or architecture (of a technical system)